Makefile

```diff
@@ -0,0 +1,10 @@
+build:
```

Comment:
nice! what about updating the ci script as well to use the make file?
https://github.com/mendixlabs/mx-docker-tools/blob/main/.github/workflows/docker-image-ci.yml#L23